assets/src/edit-story/elements/text/layer.js

```diff
@@ -36,7 +36,8 @@ const TextLayer = styled.span`
 function TextLayerContent({ element: { content } }) {
   // Remove all tags
   const rawContent = content.replace(/<[^>]*>/g, '');
-  return <TextLayer>{rawContent}</TextLayer>;
+  const ariaLabel = `Text element ("${rawContent}")`;
```

Comment:
Is this the best label here?

Reply:
RE: the line above, I did some really bad HTML stripping for `TextContent` template. We should probably move it to utils and share. And maybe even find a cheaper way to do it. The method I used is horrible, but it's 100% accurate, including things like `&lt;`.

Reply:
I can extract to a util. Where's the other instance?

Reply:
Done in 8383802.